compute/admin_guide/authentication/oidc.adoc

```diff
@@ -116,12 +115,92 @@ Configure Prisma Cloud.
 
 .. Click *Save*.
 
+[.task]
+=== Azure Active Directory (AD)
+
+To integrate with Azure Active Directory (AD), you must register Prisma Cloud as an Open ID Connect (OIDC) application in Azure and configure Azure AD as an identity provider in Prisma Cloud.
+
+[.procedure]
+
+. Go to https://portal.azure.com/#home[your Azure console].
+
+. Find the Azure AD service.
+
+. Click the *app registration* button and select *New registration*
+
+. Enter a name and select *Twistlock only - single tenant* as the supported account type.
+
+. Under *Redirect URI* select *Web platform* and enter the path to the Prisma Cloud Console API for OIDC, for example https://`<CONSOLE_URL>`:8083/api/v1/authenticate/callback/oidc
+
+. Click on *Register the app*.
+
+. To add the secret for the client, go to *certificates & secrets*.
+
+. Add a new secret for the client, copy and store it for later use.
++
+[IMPORTANT]
+====
+You can only view the value of the secret when you create it. Copy and store the secret safely for later use.
+====
+
+. To add the needed claim, go to *Token Configuration*.
+
+. Add a group claim and select the groups you want to assign to the application.
```

Comment:
select the groups you want to assign to the application -> 	select ‘groups assigned to the application’. This is a general settings.

Reply:
Thank you for clarifying. @Tomerit I'm calling out explicitly that it's a setting customers need to select.